packages/framework/ini-files/config-specs.ini

```diff
@@ -2350,6 +2350,33 @@ opt-set-cmake-var Zoltan2_scotch_example_MPI_4_DISABLE BOOL : ON
 
 use RHEL7_POST
 
+[rhel7_sems-intel-2021.3-sems-openmpi-4.0.5_release-debug_shared_no-kokkos-arch_no-asan_no-complex_fpic_mpi_no-pt_no-rdc_no-uvm_deprecated-on_no-package-enables]
+use RHEL7_SEMS_COMPILER|INTEL
+use BUILD-TYPE|RELEASE-DEBUG
+use RHEL7_SEMS_LIB-TYPE|SHARED
+use KOKKOS-ARCH|NO-KOKKOS-ARCH
+use USE-ASAN|NO
+use USE-COMPLEX|NO
+use USE-FPIC|YES
+use USE-MPI|YES
+use USE-PT|NO
+use USE-RDC|NO
+use USE-UVM|NO
+use USE-DEPRECATED|YES
+use PACKAGE-ENABLES|NO-PACKAGE-ENABLES
+use COMMON
+
+opt-set-cmake-var CMAKE_CXX_STANDARD STRING : 17
+opt-set-cmake-var ML_ENABLE_SuperLU BOOL FORCE : OFF
+opt-set-cmake-var TPL_Netcdf_LIBRARIES STRING FORCE : -L${NETCDF_C_LIB|ENV}/lib;${NETCDF_C_LIB|ENV}/libnetcdf.so;${PARALLEL_NETCDF_LIB|ENV}/libpnetcdf.a
+opt-set-cmake-var TPL_HDF5_LIBRARIES STRING FORCE : ${HDF5_LIB|ENV}/libhdf5_hl.so;${HDF5_LIB|ENV}/libhdf5.a;${ZLIB_LIB|ENV}/libz.a;-ldl
+opt-set-cmake-var MueLu_UnitTestsEpetra_MPI_1_DISABLE BOOL FORCE : ON
```

Comment:
@jmlapre: Please link to the related github issue that reports these test failures to MueLu.

Reply:
I'm working through it.  Many of the disabled tests can be re-enabled as the related testcases are now passing.  Down to the last few now...